src/onnxruntime.cc

```diff
@@ -1547,6 +1547,9 @@ ModelInstanceState::ProcessRequests(
     if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
       preferred_memory_type = TRITONSERVER_MEMORY_GPU;
       preferred_memory_type_id = DeviceId();
+    } else {
+      preferred_memory_type = TRITONSERVER_MEMORY_CPU;
+      preferred_memory_type_id = 0;
```

Comment:
You don't need to add this, in line 1545 the values have been initialized to CPU / 0